azext_iot/monitor/parsers/common_parser.py

```diff
@@ -28,18 +30,29 @@ def __init__(self, message: Message, common_parser_args: CommonParserArguments):
         self._message = message
         self.device_id = ""  # need to default
         self.device_id = self._parse_device_id(message)
-        self.interface_name = self._parse_interface_name(message)
         self.module_id = self._parse_module_id(message)
+        self.interface_name = self._parse_interface_name(message)
+        self.component_name = self._parse_component_name(message)
+
+    def parse_message(self, **kwargs) -> dict:
```

Comment:
can we plumb the args through the common_parser_args object instead? I wanted parse_message to never have anything passed to it

Reply:
can we remove **kwargs from this now that we arent passing anything into it anymore

Reply:
removed